zooniverse/front-end-monorepo
# Update ZooFooter links to reflect new FEM pages

❌ Do not merge until launch of new About and Get Involved pages.

This corresponds to https://github.com/zooniverse/Panoptes-Front-End/pull/7157.

## Package
lib-react-components

## Linked Issue and/or Talk Post
Closes: https://github.com/zooniverse/front-end-monorepo/issues/6025

## Describe your changes
- Update ZooFooter links to match [Figma](https://www.figma.com/design/6rXVi6arrGTmoM08x9tpex/UI-Kit?node-id=5-409&m=dev&t=MGdDZm4sLEz9aSXP-1) and link to new FEM pages
- I had to edit several translations dictionaries, so I'll follow-up this PR with related changes in Lokalise

- Also snuck in an update from twitter logo to X logo in both the ZooFooter and ConnectWithProject.

## How to Review
- Run lib-react-components Storybook to see the ZooFooter
- Run app-root or app-project locally and visit any page

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## Post-merge
